docs/sources/administration/api-keys/index.md

```diff
@@ -27,11 +27,19 @@ Create an API key when you want to manage your computed workload with a user.
 
 This topic shows you how to create an API key using the Grafana UI. You can also create an API key using the Grafana HTTP API. For more information about creating API keys via the API, refer to [Create API key via API]({{< relref "../../developers/http_api/create-api-tokens-for-org/#how-to-create-a-new-organization-and-an-api-token" >}}).
 
-### Before you begin:
+### Before you begin
+
+To follow these instructions, you need one of the following:
```

Comment:
```suggestion
To follow these instructions, you need at least one of the following:
```

Reply:
it is true that you only need at least one